src/pages/Architecture/index.js

```diff
@@ -92,23 +97,41 @@ function ArchitecturesSection() {
 
     useEffect(() => {
         if (initialized) {
-            fetch(process.env.REACT_APP_API_BASE_URL + '/api/projects/architectures/' + parentId, {
-                method: 'get',
-                headers: {
-                    'Content-Type': 'application/json',
-                    Authorization: initialized ? `Bearer ${keycloak?.token}` : undefined,
-                },
-            })
-                .then(response => response.json())
-                .then(result => {
-                    if (result?.data) {
-                        const archslist = structuredClone(result.data);
-
-                        setArchitectures(archslist);
-                        setTotalArchitectures(archslist.length);
-                    }
+            if (keycloak?.tokenParsed?.given_name === 'Admin' && location.pathname === '/architectures') {
```

Comment:
We can go with the give_name then only the person with the name admin can only create the projects